Automattic/simplenote-ios
# Shortcuts update part 4: reimplement open note with siriintents

### Fix
Next step in implementing shortcuts with siri kit is to implement the open note shortcut.  This allows you to open into Simplenote in any previously selected note.

fixes #1568 
### Test
1. Run the branch, go to the shortcuts app, create a new shortcut, add action, find Simplenote and select the open note shortcut. 
2. Pick a note to open
3. run the shortcut and confirm that Simplenote opens that note

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
> These changes do not require release notes.
